tests/integration/pilot/testdata/upgrade/1.8.0-install.yaml

```diff
@@ -262,7 +262,7 @@ spec:
             filter:
               name: "envoy.http_connection_manager"
               subFilter:
-                name: "envoy.router"
+                name: "envoy.filters.http.router"
```

Comment:
this is intentionally a snapshot of the old install charts to test upgrade so it shouldn't change

Reply:
Will revert